src/lib.rs

```diff
@@ -59,6 +60,8 @@ pub struct DetectionResults {
     pub scene_changes: Vec<usize>,
     /// The total number of frames read.
     pub frame_count: usize,
+    /// Average speed (FPS)
+    pub speed: f64,
```

Comment:
What is the use case for having av-scenechange itself measure this? It doesn't seem like the CLI prints this info. Is there a benefit to doing this here rather than having whatever library is using av-scenechange calculate this?